api/v1/ptpconfig_webhook.go

```diff
@@ -40,7 +40,7 @@ const (
 
 // log is for logging in this package.
 var ptpconfiglog = logf.Log.WithName("ptpconfig-resource")
-var profileRegEx = regexp.MustCompile(`^(\w+)(,\s*([\w-_]+))`)
+var profileRegEx = regexp.MustCompile(`^([\w\-_]+)(,\s*([\w-_]+))`)
```

Comment:
This is wrong, should be `^([\w\-_]+)(,\s*([\w\-_]+))`. Note the second `-` should have an escape `\`.

Reply:
In Go, the underscore character _ does not require escaping when used in regular expressions.

Reply:
I was referring to the dash, not underscore